tools/c7n_gcp/c7n_gcp/region.py

```diff
@@ -48,6 +49,13 @@ def set_regions(cls, regions):
     def resources(self, resource_ids=()):
         if resource_ids:
             return [{'name': r} for r in self.regions if r in resource_ids]
+        elif self.config.regions or self.config.region != 'us-east-1':
+            regions = list(self.config.regions)
+            regions.append(self.config.region)
+            regions = list(filter(None, regions))
+            if 'us-east-1' in regions:
+                regions.remove('us-east-1')
```

Comment:
I'm not sure I follow this in the case where a run specifies multiple regions and one is us-east-1, but I'm implicitly trusting it 😅

Reply:
the logic here is just to handle the default in the empty config, per https://github.com/cloud-custodian/cloud-custodian/blob/main/c7n/config.py#L31